src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java

```diff
@@ -0,0 +1,130 @@
+                    ReplyKeyboardMarkup replyKeyboardMarkup = new ReplyKeyboardMarkup();
+                    replyKeyboardMarkup.setResizeKeyboard(true);
+                    replyKeyboardMarkup.setSelective(true);
+
+                    List<KeyboardRow> keyboardRowList = new ArrayList<>();
+                    KeyboardRow keyboardRow1 = new KeyboardRow();
+                    KeyboardRow keyboardRow2 = new KeyboardRow();
+
+                    KeyboardButton keyboardButton1 = new KeyboardButton();
+                    keyboardButton1.setText("منوی برگزار کننده");
+
+                    KeyboardButton keyboardButton2 = new KeyboardButton();
+                    keyboardButton2.setText("منوی شرکت کننده");
+
+                    keyboardRow1.add(keyboardButton1);
+                    keyboardRow2.add(keyboardButton2);
+
+                    keyboardRowList.add(keyboardRow1);
+                    keyboardRowList.add(keyboardRow2);
+
+                    replyKeyboardMarkup.setKeyboard(keyboardRowList);
+                    sendMessage.setReplyMarkup(replyKeyboardMarkup);
+
+
+                    try {
+                        execute(sendMessage);
+                    } catch (TelegramApiException e) {
+                        throw new RuntimeException(e);
+                    }
+                } else if (message.getText().equals("منوی برگزار کننده")) {
+                    SendMessage sendMessage = new SendMessage();
+                    sendMessage.setText("خدمات مربوط به برگزاری ایونت");
+                    sendMessage.setChatId(message.getChatId());
+                    sendMessage.setParseMode(ParseMode.MARKDOWN);
+
+                    // InlineKeyboardButton
+                    InlineKeyboardMarkup inlineKeyboardMarkup = new InlineKeyboardMarkup();
+
+                    List<List<InlineKeyboardButton>> inlineButtons = new ArrayList<>();
+                    List<InlineKeyboardButton> inlineKeyboardButtonList1 = new ArrayList<>();
+                    List<InlineKeyboardButton> inlineKeyboardButtonList2 = new ArrayList<>();
+
+                    InlineKeyboardButton inlineKeyboardButton1 = new InlineKeyboardButton();
+
+                    inlineKeyboardButton1.setText("ايجاد يك رویداد جدید");
+
+                    inlineKeyboardButton1.setCallbackData("ايجاد يك رویداد جدید");
+
+                    inlineKeyboardButtonList1.add(inlineKeyboardButton1);
+
+                    inlineButtons.add(inlineKeyboardButtonList1);
+                    inlineButtons.add(inlineKeyboardButtonList2);
+
+                    inlineKeyboardMarkup.setKeyboard(inlineButtons);
+                    sendMessage.setReplyMarkup(inlineKeyboardMarkup);
+
+                    try {
+                        execute(sendMessage);
+                    } catch (TelegramApiException e) {
+                        e.printStackTrace();
+                    }
+                }
+
+            }
+        } else if (update.hasCallbackQuery()) {
```

Comment:
What is call back query? Why we need to check it here?

Reply:
> What is call back query? Why we need to check it here?

https://core.telegram.org/bots/api#callbackquery

Reply:
In Telegram bots, a CallbackQuery is a type of message that represents a user's interaction with an inline keyboard. When a user presses a button in an inline keyboard, a callback query is generated and sent to the bot, containing information about the button that was pressed.

Callback queries are useful for creating interactive and dynamic bot interfaces, allowing users to trigger specific actions or provide input without having to send a separate message.